libamqpprox/amqpprox_connector.cpp

```diff
@@ -298,9 +298,12 @@ void Connector::synthesizeCloseError(bool sendToIngressSide)
     synthesizeMessage<Reply::CloseOkExpected>(d_close, sendToIngressSide);
 }
 
-void Connector::synthesizeCloseAuthError(bool sendToIngressSide)
+void Connector::synthesizeCustomCloseError(bool     sendToIngressSide,
+                                           uint16_t code,
+                                           const std::string_view &text)
```

Comment:
```suggestion
                                           std::string_view text)
```

I believe it's normal to pass `string_view` by value since copying them is very cheap.